forte/processors/data_augment/algorithms/eda_processors.py

```diff
@@ -0,0 +1,257 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Data augmentation processors from the paper "EDA: Easy Data Augmentation
+Techniques for Boosting Performance on Text Classification Tasks", including
+Random Swap, Random Insertion and Random Deletion. All three processors are
+implemented based on the ReplacementDataAugmentProcessor.
+"""
+
+from math import ceil
+import random
+from typing import List, Dict
+
+from forte.common.configuration import Config
+from forte.common.resources import Resources
+from forte.data.data_pack import DataPack
+from forte.data.multi_pack import MultiPack
+from forte.processors.base import ReplacementDataAugmentProcessor
+from forte.utils.utils import get_class, create_class_with_kwargs
+
+__all__ = [
+    "RandomSwapDataAugmentProcessor",
+    "RandomInsertionDataAugmentProcessor",
+    "RandomDeletionDataAugmentProcessor",
+]
+
+
+# pylint: disable=W0201
```

Comment:
Now in pylint we can use the text form o the message. It is easier to follow.

Reply:
Should I put it in the `default_configs` function of `RandomInsertionDataAugmentProcessor`. I didn't do it because it's making the function too long and hard to read.

Reply:
Yeah, you can put it into the function where this pylint warning happens.